src/test/scala/HttpVerticleSpec.scala

```diff
@@ -1,20 +1,24 @@
+import org.junit.runner.RunWith
 import org.scalatest.Matchers
-
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.junit.JUnitRunner
 import scala.concurrent.Promise
 
-class HttpVerticleSpec extends VerticleTesting[HttpVerticle] with Matchers {
+@RunWith(classOf[JUnitRunner])
```

Comment:
Why are you introducing the JUnit-Dep and adding the Runner here?

Reply:
See http://www.scalatest.org/user_guide/using_junit_runner. This allows older versions of Scala IDE to run ScalaTest

Reply:
Do you have a pointer to what versions of ScalaIDE are affected (I am an Idea user so I have no clue about its limitations).
I am asking because I am very cautious with introducing additional libraries. I haven't used the JUnitRunner in Scala-projects in the past 2 years.
This dependency would be included for every user and if the problem affects only a small user bases using an old (outdated?) IDE-version then I'd rather go without it.

And thanks for your contribution :) 
A nice improvement of the current template.